index.js

```diff
@@ -0,0 +1,13 @@
+var express = require( "express" );
+var app, server, port;
+
+//jscs:disable requireCamelCaseOrUpperCaseIdentifiers
+port = process.env.npm_package_config_port;
+
+//jscs:enable requireCamelCaseOrUpperCaseIdentifiers
+
+app = express();
+app.use ( "/", express.static( "./static" ) );
+server = app.listen( port, function() {
+	console.log( "Listening on " + port + "..." );
```

Comment:
You don't have to specify the host when calling `listen()`, so just output localhost, no need to make that configurable.